recipe/meta.yaml

```diff
@@ -99,8 +94,6 @@ outputs:
             # shared
             - test -f $PREFIX/lib/libkaldi-{{ each_lib }}.so               # [linux]
             - test -f $PREFIX/lib/libkaldi-{{ each_lib }}.dylib            # [osx]
-            # windows doesn't yet build dynamic libs
-            - if not exist %LIBRARY_BIN%\kaldi-{{ each_lib }}.dll exit 1   # [win]
```

Comment:
Sorry, I meant that the comment `# windows doesn't yet build dynamic libs` was outdated, we should leave the test of course!

Reply:
No my bad, thanks for catching that!